feature-libs/product-configurator/rulebased/components/attribute/delta-rendering/configurator-delta-rendering.service.ts

```diff
@@ -0,0 +1,118 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { inject, Injectable } from '@angular/core';
+import { ObjectComparisonUtils } from '@spartacus/core';
+import { ConfiguratorRouterExtractorService } from '@spartacus/product-configurator/common';
+import { EMPTY, filter, Observable, of, switchMap, tap } from 'rxjs';
+import { ConfiguratorCommonsService } from '../../../core/facade/configurator-commons.service';
+import { Configurator } from '../../../core/model/configurator.model';
+
+@Injectable()
+export class ConfiguratorDeltaRenderingService {
```

Comment:
TBH I'm not fond of this service soudning very technical, related to the rendering process - like `RerenderingService` or a method `rerender`.
Going this path, we should name all services and methods that return any updated data "shouldRerender" etc.

Isn't this service simply a "Price Changes service"?

e.g. something like `ConfiguratorPriceChangeService` or `ConfiguratorAttributePriceChangeService`?
with a method `getPriceChangeEvents(attributeKey): Observable<true>` or `getPriceChangeEvents(attributeKey): Observable<{ priceChanged: boolean }>`?